cypress/integration/changeSignalAfterSubmit.spec.js

```diff
@@ -0,0 +1,324 @@
+// <reference types="Cypress" />
+import * as createSignal from '../support/commandsCreateSignal';
+import { CREATE_SIGNAL } from '../support/selectorsCreateSignal';
+import { CHANGE_CATEGORY, CHANGE_LOCATION, CHANGE_STATUS, CHANGE_TYPE, CHANGE_URGENCY, SIGNAL_DETAILS } from '../support/selectorsSignalDetails';
+import { MANAGE_SIGNALS, OVERVIEW_MAP } from '../support/selectorsManageIncidents';
+describe('Change signal after submit', () => {
+  describe('Create signal graffitie', () => {
+    before(() => {
+      cy.server();
+      cy.defineGeoSearchRoutes();
+      cy.getAddressRoute();
+
+      cy.visitFetch('incident/beschrijf');
+    });
+
+    it('Should search for an address', () => {
```

Comment:
There is a lot of repetition in tests. Take searching for an address, entering a description, phone number, email address and such. If the structure of the form changes (and it will), then all of those tests need to be revised and fixed. I would extract any repeating patterns into functions where possible.